compiler/src/main/java/com/squareup/anvil/compiler/codegen/ContributesSubcomponentHandlerGenerator.kt

```diff
@@ -101,9 +105,51 @@ internal class ContributesSubcomponentHandlerGenerator(
     module: ModuleDescriptor,
     projectFiles: Collection<KtFile>
   ): Collection<GeneratedFile> {
+    if (isFirstRound) {
```

Comment:
Nit: This `generateCode` method seems to have gotten quite large and it makes what's happening a bit harder to follow. It would be helpful to break some of its logic out to new helper methods. e.g.
```
if (isFirstRound) {
  isFirstRound = false
  populateInitialContributions(module)
  populateReplacedContributions(module)
}
```

Reply:
Good idea, done.